device/celestica/x86_64-cel_seastone-r0/port-mode.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I agree.   we could add that to the sonic-utilities/script, long term it would be good to integrate with the configDB design.  "show platform summary" should be able to print the current hwsku.